### ISSUE TITLE
[Bug Report][3.1.10] VFooter does not apply height

### ISSUE THREAD

**Issue by abea** [vuetifyjs/vuetify#16934]
### Environment
**Vuetify Version:** 3.1.10
**Vue Version:** 3.2.47
**Browsers:** Firefox 110.0
**OS:** Mac OS 10.15

### Steps to reproduce
Add a VFooter component with [`height` prop](https://vuetifyjs.com/en/api/v-footer/#props) assigned to a number or px value.

### Expected Behavior
The footer should be the height assigned.

### Actual Behavior
It uses the `'auto'` height.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdkd1OwkAQhV9l3Bs0oV38SUwIGr3TJ/CC5aK0U1jZv+xuEUN4d2dbMBUiid71zJn9zsx0umXPzuXrBtmYTSJqp4qIj8IATNZZ4Vz72QpdSLNXrY64iVktUVVApq1QPQimw0Iw4P2+2tqIHkqrrKeOuWqQWpYoF8tI+mY0chvBHl8kvEJY2oZ4c4S2DLFQasIPjDPUhUc0hB33uX+ipsL3ikl0y0947ygkQ+mlixAwNsmW2lkfYQsea9hB7a2GAV1zkJpLa0IEOgk8JP9y8IJKWXizXlUXg6sE73AEYkPWsWgIl78Ha+iHbNMsYm8EwcbQVlKNMpIWbBmjC2POG+NWi7y0mj+Rx31jotSYVVY/3eY3+d09r2SI/XqOQWdzbz8CekoUbNiDR1l//hawt4l7nV+POu6+lqliHhL4BMgpZY0+82gq9OjPTn/U+2ODI+9ki5S5E2ZHF1XSrMLRMcvQHnJ6GO0f67WI9H6WYnZD1o1Hgd0TNvsCh/QZyQ==)

**Comment by yuwu9145** [comment]
Expected behaviour can be found in  [this link](https://github.com/vuetifyjs/vuetify/issues/14983#issuecomment-1175266569) 

You should add `app` prop:

https://play.vuetifyjs.com/#eNqdkd1OwkAQhV9l3Bs0oV1EExMCBu/wCbygXJR2Civ7l90tYgjv7mwLpkIk0bvOnNnvzJzO9+zF2nRbIxuxcUBlZR7wOdMA422SW9t8NoXKhT5WTR1wF5JKoCyBRFOinGRM+VXGgHfnKmMCOiiMNI4mlrJGGlmjWK0D1cPBwO6oEb1mAl7Br01NzCVCI0HIpRzzE+cKeeUQNZFGXfafybHxfWos2hDGvBMOlb5wwgbwGOooC2WNC7AHhxUcoHJGQY9S7cXhwmgfgKKBSdRvezOU0sCbcbK86d1FeIsjEOuzlkVL2PTdG00/Zh93yY6Cz9gImk7skUesM7YOwfoR57W2m1VaGMWnpHFX6yAUJqVR04d0mD4+8VL40O2n6FWydObDoyPHjPU78CCqz98MjjJx79P7Qcs99hKZL30EXwA5uWzRJQ51iQ7d1e3PZn9ccKZdXBE9D5k+UKJS6I0/C7PwTZDz02r/OK9BxPeLaHPos3Y9MmyfsMUXnscciw==

**Comment by abea** [comment]
What's actually happening there? Neither the `app` or `height` props have descriptions in the docs. I mostly want to make sure I understand if there are any other effects of that prop.

**Comment by yuwu9145** [comment]
Docs are "working in progress". Before it's fully ready, `app` prop on VFooter should be working in a similar way as in v2 ([link](https://v2.vuetifyjs.com/en/api/v-footer/#props)).

It's also helpful to check out [v3 application-layout](https://vuetifyjs.com/en/features/application-layout/) combined with  [examples of how to do this now](https://github.com/vuetifyjs/vuetify/issues/14983#issuecomment-1175266569)

Basically  `app` prop must be set to be able to get "layout-related" styles including `height` working.

**Comment by abea** [comment]
Thanks for the follow up. In the documentation I would suggest elaborating on what "layout-related" includes. I can understand the team's choice to be more opinionated with those props, though it does have the effect of requiring more custom overrides to get a similar outcome now.
